src/GraphQL/Execution/DocumentExecuter.cs

```diff
@@ -251,6 +250,18 @@ public object GetProperyValue(object obj, string propertyName)
 
             fields.Apply(field =>
             {
+                if (objectType is IReturnsAllFields)
```

Comment:
Having this does seem to go against the idea of GraphQL, in that you only request what you need.  If you do need "everything", dynamically, you could use the introspection pieces to find out what all possible fields are.